packages/transaction-backend/.env.example

```diff
@@ -1,2 +1,16 @@
-RPC_URL=https://dev-ethereum-node.rarible.com
-SDK_ENV=development
+
+PORT=5500
+RARIBLE_API_KEY=4df39770-4e14-42ce-87d8-27e6d0b68167
+
+# "mainnet" | "testnet" | "staging" | "mumbai" | "polygon" | "dev-ethereum" | "dev-polygon" | "staging-polygon" | "mantle" | "testnet-mantle"
+POLYGON_SDK_ENV=polygon
+POLYGON_RPC_URL=https://polygon-mainnet.infura.io/v3/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+ETEHREUM_SDK_ENV=mainnet
+ETEHREUM_RPC_URL=https://ethereum-mainnet.infura.io/v3/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+MANTLE_SDK_ENV=mantle
+MANTLE_RPC_URL=https://mantle-mainnet.infura.io/v3/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# base URL to make calls to the Rarible API
+RARIBLE_BASE_PATH=
```

Comment:
it should be assigned in runtime and depends on env

Reply:
why in runtime? Usually RPC url on start server